web_src/js/features/citation.js

```diff
@@ -0,0 +1,68 @@
+import $ from 'jquery';
+
+const {pageData} = window.config;
+
+const initInputCitationValue = async () => {
+  const [{Cite, plugins}] = await Promise.all([
+    import(/* webpackChunkName: "citation-js-core" */'@citation-js/core'),
+    import(/* webpackChunkName: "citation-js-formats" */'@citation-js/plugin-software-formats'),
+    import(/* webpackChunkName: "citation-js-bibtex" */'@citation-js/plugin-bibtex'),
+    import(/* webpackChunkName: "citation-js-bibtex" */'@citation-js/plugin-csl'),
+  ]);
+  const {citiationFileContent} = pageData;
+  const $citationCopyApa = $('#citation-copy-apa');
+  const $citationCopyBibtex = $('#citation-copy-bibtex');
+  const config = plugins.config.get('@bibtex');
+  config.constants.fieldTypes.doi = ['field', 'literal'];
+  config.constants.fieldTypes.version = ['field', 'literal'];
+  const citationFormatter = new Cite(citiationFileContent);
+  const apaOutput = citationFormatter.format('bibliography', {
+    template: 'apa',
+    lang: 'en-US'
+  });
+  const bibtexOutput = citationFormatter.format('bibtex', {
+    lang: 'en-US'
```

Comment:
If these plugins are support other languages, we could obtain the language dynamically from `document.documentElemen.lang` (`lang` attribute on `<html>` that switches with language setting).